src/sdk-reference/js/6/offline-tools/index.md

```diff
@@ -10,66 +10,88 @@ order: 400
-Works only in `offline` state when the `autoQueue` option is set to `false`. Call `playQueue` to send to Kuzzle the
-requests in the queue, once the SDK state passes to `online`. Call `flushQueue` to empty the queue without sending the requests.
+A read-only `Array` containing the requests queued while the SDK is in the `offline` state (it behaves like a FIFO queue).
 
-## stopQueuing()
+### queueMaxSize
 
-Stop queuing the requests. Requests will no more be put in the `offlineQueue`, they will be discarded.
-Works only in the `offline` state, and if the `autoQueue` option is set to `false`.
+A writable `number` defining the maximun size of the `offlineQueue`.
 
-## playQueue()
+Default value: `500`
 
-Sends to Kuzzle all the requests in the `offlineQueue`. Works only if the SDK is not in a `offline` state, and if the 
-`autoReplay` option is set to false.
+### queueTTL
 
-## flushQueue()
+A writable `number` defining the time in milliseconds a queued request is kept in the `offlineQueue`.
 
-Empties the `offlineQueue` without sending the requests to Kuzzle.
+Default value: `120000`
 
-## autoQueue
+### autoQueue
 
 A writable `boolean` telling the SDK whether to automatically queue requests during the `offline` state or not.
 
-## autoReplay
+Default value: `false`
+
+### autoReplay
 
 A writable `boolean` telling the SDK whether to automatically send or not the requests in the `offlineQueue` on a
 `reconnected` event.
 
-## autoReconnect
+Default value: `false`
+
+### autoReconnect
 
 A writable `boolean` telling the SDK whether to automatically reconnect or not to Kuzzle after a connection loss.
 
-## reconnectionDelay 	
+Default value: *Depends on the Protocol*
+
+### reconnectionDelay 	
 
 A read-only `number` specifying the time in milliseconds between different reconnection attempts.
 
-## autoResubscribe
+Default value: *Depends on the Protocol*
+
+### autoResubscribe
 
 A writable `boolean` telling the SDK whether to automatically renew or not all subscriptions on a reconnected event.
 
-## queueFilter
+Default value: `true`
+
+### queueFilter
 
 A writable `Function` called by the SDK each time a `Request` need to be queued. The `Request` is passed as the only argument
 to the function and is queued only if the function returns `true`. Use it to define which requests are allowed to be queued.
 
-## offlineQueueLoader
+### offlineQueueLoader
 
 A writable `Function` called by the SDK before playing the requests in the `offlineQueue`. This function takes no arguments
 and returns an array of `Request` that are added on top of the `offlineQueue`. Use it to inject new requests to be played
 before the queue.
+
+## Methods
+
+### [startQueuing()]({{ site_base_path }}sdk-reference/js/6/kuzzle/start-queuing/)
+
+Starts queuing requests when in `offline` state. Request will be put in the `offlineQueue` instead of being discarded, until `stopQueuing` is called.
```

Comment:
```suggestion
Starts queuing requests when in `offline` state. Requests will be put in the `offlineQueue` array instead of being discarded, until `stopQueuing` is called.
```